cmd/lytfs/main.go

```diff
@@ -31,9 +34,12 @@ func main() {
 	ctx, cancel := context.WithTimeout(context.Background(), *timeout)
 	defer cancel()
 
+	owner := strings.Split(*repo, "/")[0]
+	repo := strings.Split(*repo, "/")[1]
```

Comment:
I think this can be optimized to not call `strings.Split` twice — let me know what you prefer.